### PR TITLE
[WFCORE-6094] Increase LayersTest timeout

### PR DESCRIPTION
**Pull request by lvydra**
https://issues.redhat.com/browse/WFCORE-6094

PR depends on https://github.com/wildfly/wildfly/pull/16242

### CODE REVIEW

File: testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java [hunk 1]
```diff
@@ -208,7 +208,7 @@ public void run() {
             }
         };
         try {
-            executor.submit(r).get(1, TimeUnit.MINUTES);
+            executor.submit(r).get(2, TimeUnit.MINUTES);
```
Comment (yersan) [inline testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java:211]: Hello @lvydra , would you mind using [TimeoutUtil.java](https://github.com/wildfly/wildfly-core/blob/main/testsuite/shared/src/main/java/org/jboss/as/test/shared/TimeoutUtil.java) for handling this timeout?

This utility class allows timeouts to be modified with a correction factor. Slow CI environments can append the `ts.timeout.factor` property to increase the default timeout, making the testsuite more reliable in slow environments.

Thanks

**Review by yersan — CHANGES_REQUESTED** [review]

**Review by yersan — CHANGES_REQUESTED** [review]
Sadly this is going to need changes in wildfly full before merging this. The test case is complaining about a missing dependency for the test scope for the testsuite/layers module:

org.wildfly.security.manager.WildFlySecurityManager

**Comment by lvydra** [comment]
Hi @yersan, I have opened PR in wildfly which adds the dependency.

**Review by yersan — APPROVED** [review]
Thanks @lvydra , it is green now

**Pull request merged**